addons/mrp/models/stock_rule.py

```diff
@@ -75,6 +75,8 @@ def _run_manufacture(self, procurements):
             if not mo:
                 procurement_qty = procurement.product_qty
                 batch_size = procurement.values.get('batch_size', procurement_qty)
+                if not batch_size or batch_size <= 0:
```

Comment:
```suggestion
                if batch_size <= 0:
```
`not batch_size` is already covered by the default value of the `get()` on the previous line.
Hopefully the `procurement_qty` will never be negative, right? :eyes:

Reply:
done, but you never know, maybe the next python version won't consider None as False anymore?